code/__DEFINES/subsystems.dm

```diff
@@ -200,6 +201,8 @@
 #define FIRE_PRIORITY_EXPLOSIONS	666
 #define FIRE_PRIORITY_TIMER			700
 #define FIRE_PRIORITY_SOUND_LOOPS	800
+#define FIRE_PRIORITY_SPEECH_CONTROLLER 900
```

Comment:
What is the difference between this and FIRE_PRIORITY_CHAT?

Reply:
FIRE_PRIORITY_CHAT handles the tgui aspect of chat, the controller handles the physical say and message call which before was unmanaged